api/config.pb.go

```diff
@@ -1,6 +1,6 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.23.0
+// 	protoc-gen-go v1.25.0-devel
```

Comment:
We should not use a `devel` version here, I guess. Also, we should add some kind of automation to prevent us from using different `protoc-gen-go` versions.

Reply:
don't know how I had that devel version...🤷‍♂️